mutacc/utils/bam_handler.py

```diff
@@ -169,23 +153,34 @@ def find_reads_from_region(self, chrom, start, end):
                 self.reads.pop(key)
                 self.found_reads = self.found_reads.union({key})
 
-            #AlignmentSegment.mate raises ValueError if mate can not be found
-            #If mate is not found, the single read is not added to the out_bam file
-            #(Unless ends argument in __init__ is not set to 1)
-            except ValueError:
-                LOG.warning("Mate not found for read {}, {}\n mate is unmapped: {}".format(
-                    key,
-                    self.reads[key][0].next_reference_id,
-                    self.reads[key][0].mate_is_unmapped
-                    )
-                )
 
 
     @property
     def record_number(self):
 
         return len(self.found_reads)
 
+    def find_mate(self, read):
+
```

Comment:
Could you add proper doc strings to all your defs in another PR

Reply:
absolutely